gammapy/image/models/core.py

```diff
@@ -110,6 +111,85 @@ def evaluate(lon, lat, lon_0, lat_0, sigma):
         return norm * np.exp(exponent)
 
 
+class SkyElongatedGaussian(SkySpatialModel):
+    r"""Two-dimensional elongated Gaussian model (defined on the plane).
+
+    .. math::
+
+             f(\text{lon}, \text{lat}) = \frac{1}{2\pi\sigma_{\text{lon}}\sigma_{\text{lat}}} e^{-a\left(\text{lon} - \text{lon}_{0}\right)^{2}  -b\left(\text{lon} - \text{lon}_{0}\right)\left(\text{lat} - \text{lat}\right)  -c\left(\text{lat} - \text{lat}_{0}\right)^{2}}
+
+    Using the following definitions:
+
+            a = \left(\frac{\cos^{2}{\left (\theta \right )}}{2 \sigma_{\text{lon}}^{2}} +
+            \frac{\sin^{2}{\left (\theta \right )}}{2 \sigma_{\text{lat}}^{2}}\right)
+
+            b = \left(\frac{\sin{\left (2 \theta \right )}}{2 \sigma_{\text{lon}}^{2}} -
+            \frac{\sin{\left (2 \theta \right )}}{2 \sigma_{\text{lat}}^{2}}\right)
+
+            c = \left(\frac{\sin^{2}{\left (\theta \right )}}{2 \sigma_{\text{lon}}^{2}} +
+            \frac{\cos^{2}{\left (\theta \right )}}{2 \sigma_{\text{lat}}^{2}}\right)
+
+    where :math:`\sigma_{\text{lon}}` and :math:`\sigma_{\text{lat}}` are the 68% containment semi-axes of the gaussian before rotation, and :math:`\theta` is the rotation angle (with respect to the constant latitude axis).
```

Comment:
Same here, please break the line to stick within the 80 character limit.

I don't think sigma values for 2D have something to do with 68% containment, suggest to just remove that statement.